### ISSUE TITLE
PostgreSQL 10

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse-ansible#63]
At standup we just talked about https://github.com/IQSS/dataverse/issues/5809 and the note from @donsizemore that he's been playing around with running Dataverse on PostgreSQL 10 and 11. This is exactly the sort of innovation we have come to love from @donsizemore such as how he forged the way toward Shibboleth 3. He also added support for changing Java versions in this dataverse-ansible repo so that we can experiment with Java 11 some day. In that spirit, it would be great if dataverse-ansible supported the ability to switch between PostgreSQL 9 and 10 (and 11 I guess, but it isn't yet supported by Flyway).

I found this in the code:

```
  postgres:
    reporpm: https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-centos96-9.6-3.noarch.rpm
    version: 9.6
```

I assume the `reporpm` needs to change to the "latest" one @donsizemore found in the other issue? And probably other stuff. Drivers maybe. I'm not sure.

In short, we'd like to use the ec2 spin up scripts to experiment with versions of PostgreSQL beyond version 9 some day. Thanks!

**Comment by donsizemore** [comment]
This is supported via group_vars.
